packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -817,7 +824,7 @@ QUnit.module('submit element', {}, () => {
 
             assert.equal($hiddenInput.val(), '12|25', 'the correct decimal separator is used');
         } finally {
-            config(originalConfig);
+            executeActionWithSuppressErrors(() => config(originalConfig));
```

Comment:
Let's leave it as it is because the warning here is legal. 
It's also a good idea to add a check that warning is logged.

Reply:
test 'deprecated separator warning should be logged on config change' added